scripts/zones/Wajaom_Woodlands/mobs/Gharial.lua

```diff
@@ -9,6 +9,24 @@ entity.onMobInitialize = function(mob)
     mob:setMod(xi.mod.MOVE_SPEED_STACKABLE, 12)
 end
 
+entity.onMobSpawn = function(mob)
+    mob:addListener('WEAPONSKILL_STATE_EXIT', 'USE_BOILING_BLOOD', function(mobArg, skillId)
+        local roarAbility  = 2101
+        local boilingBlood = 2102
+
+        if
+            mob:getLocalVar('abilityUsed') == 0 and
```

Comment:
I dont think this variable is needed at all, considering its always set to 0 on ability use.

Reply:
It looks like.. this is logic to force use of Boiling Blood after Demoralizing Roar immediately, looks used to me. It might not act correctly after Stuns and such but it's better than not working at all.

Reply:
Correct. This mob has only 1 skill in their mobskill, forbidding it from using the second skill randomly.

However, precisely becouse of that, the use of a local variable seems unnecesary.